tests/integration/helm/upgrade/util.go

```diff
@@ -115,6 +125,14 @@ func deleteIstio(cs cluster.Cluster, h *helm.Helm) error {
 		return fmt.Errorf("wating for istio namespace to be deleted: %v", err)
 	}
 
+	// CRDs are not removed as part of the helm uninstall process
+	execCmd := fmt.Sprintf("kubectl delete -n %d -f %d", helmtest.IstioNamespace,
```

Comment:
I don't think we should recommend this or do it in tests, this will make pretty large breaking changes to users clusters. This is a bad in docs (wipe out etcd essentially), and also annoying or even dangerous in tests (when running locally especially)

Reply:
You are correct. People testing this in their clusters could also get bad behavior (e.g. OpenShift) if run locally. It's not a good practice and I've removed this line. The docs don't have this at all either. They do say to make a backup, but not to do a delete on any CRDs.